kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/model/networking/NetworkingPlanner.java

```diff
@@ -183,4 +218,28 @@ public int numIdentifyingPortsRequired() {
         }
 
     }
+
+    private record LoadBalancerClusterIngressNetworkingDefinition(
+                                                                  KafkaProxyIngress ingress,
+                                                                  VirtualKafkaCluster cluster,
+                                                                  LoadBalancer loadBalancer,
+                                                                  Tls tls)
+            implements ClusterIngressNetworkingDefinition {
+
+        private LoadBalancerClusterIngressNetworkingDefinition {
+            Objects.requireNonNull(tls);
```

Comment:
We should really be throwing an `IngressPlanningException` exception here, right?